### PR TITLE
fix: homebrew toolbelt and remove EOLed PHP

### PR DESCRIPTION
**Pull request by mdt2**
### Description

- Fixes homebrew path/name for heroku and uses heroku's tap to install toolbelt
- Removes and uninstalls dead versions of PHP

### Test

Run the laptop script to test.

### CODE REVIEW

File: php.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-php_versions=("5.6" "7.0" "7.1" "7.2")
+php_versions=("7.1" "7.2")
```
Comment (cromwellryan) [inline php.sh]: We should probably also uninstall 5.6 and 7.0 near L169.

File: mac [hunk 2]
```diff
@@ -190,7 +190,8 @@ brew cask install firefox
 brew cask install google-chrome
 brew cask install sequel-pro
 brew cask install 1password
-brew_install_or_upgrade 'heroku-toolbelt'
+brew tap heroku/brew/heroku
```
Comment (cromwellryan) [inline mac]: Let's split this into another commit.

**Comment by yosevukilonzo** [comment]
@mdt2, did you run into this error: `Error: Invalid tap name 'heroku/brew/heroku'`?
